scripts/esm/grid.mjs

```diff
@@ -320,6 +320,10 @@ class Grid extends Script {
     get resolution() {
         return this._resolution;
     }
+
+    destroy() {
+        this.entity.render.meshInstances = [];
```

Comment:
If the script adds a render component, shouldn't destroy leave the entity in the state it was beforehand? In other words, shouldn't `removeComonent('render')` be done here instead?

Reply:
Actually, to be precise, the script adds a render component if there wasn't one already. So maybe this should remove it if it was responsible for adding it.